components/suggest/src/db.rs

```diff
@@ -423,8 +426,7 @@ impl<'a> SuggestDao<'a> {
                     ":record_id": record_id.as_str(),
                     ":title": common_details.title,
                     ":url": common_details.url,
-                    ":score": common_details.score,
-
+                    ":score": common_details.score.unwrap_or(DEFAULT_SUGGESTION_SCORE)
```

Comment:
So I tried a couple things with the schema `score REAL DEFAULT 0.2` it was unhappy with me try to insert a NULL value. 
I landed on `score REAL NOT NULL ON CONFLICT REPLACE DEFAULT 0.2` and i was uncertain of that too

Reply:
Right, `score DEFAULT 0.2` wont work because we always specify `score` in `INSERT`. This works equally fine 👍